package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless",
-  "version": "0.5.5",
+  "version": "1.0.0",
```

Comment:
Finally! Maybe also worth to check out if the description etc. should also be updated